keyboards/massdrop/ctrl/keymaps/endgame_mac/rules.mk

```
@@ -0,0 +1,16 @@
+# RGBLIGHT_ENABLE = no				# Not for MD boards. This is here in case you forget.
+COMMAND_ENABLE = yes				# Commands for debug and configuration
+# BOOTMAGIC_ENABLE = full			# Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

---

Fixed this one.